### ISSUE TITLE
[Bug]: Loosing position of audiobook.

### ISSUE THREAD

**Issue by woolmonkey** [advplyr/audiobookshelf#2039]
### Describe the issue

I know this was an issue before and it seemed to be fixed for awhile, but now when using android auto I am loosing position of the book.  Even when streaming the book I stop the vehicle and come back and it starts over from where I was previously.

### Steps to reproduce the issue

1. Listen to book with android auto
2. turn off vehicle
3. restart listening to book.

### Audiobookshelf version

2.3.3

### How are you running audiobookshelf?

Docker

**Comment by NickBlomberg** [comment]
I'm not familiar with Android Auto, but I encountered a similar issue while using the Android app (version 0.9.65-beta). I tried to pause my stream after 30 minutes, but the timer had not progressed much. When I resumed the audiobook, it had jumped back to an earlier point in the chapter, not where I left off.

I suspected it might be related to battery optimization settings. So, I changed the app's battery usage to 'unrestricted.' Since making this adjustment, I haven't experienced this issue in the past three days.

Do you have any similar settings in Android Auto?

**Comment by cicavey** [comment]
I have also noticed this behavior typically if paused externally (ex. bluetooth ear buds or bluetooth pairing to car). I had not considered the background battery optimization and will try this workaround. It makes sense as most of my usage involves the the phone being left idle while listening. I also exclusively use downloaded audio books on the android app.

**Comment by valdearg** [comment]
Subscribing to this as I've really randomly experienced something similar with Android Auto.

If I turn off the car and check the "history" of the listening, I can see that there's been a "seek" command to 15 minutes ago.

I tried pausing it via the car screen then turned off the car, it sent the seek command to 15 mins ago then also.

It's done this about three times now in a row. I'm going to see if the battery optimisation helps next time I'm out.

**Comment by NickBlomberg** [comment]
I initially thought that disabling battery optimization for the app had resolved the issue, as I didn't encounter it for a while. However, I've recently experienced the problem again. Specifically, when I pause the audiobook using the lock screen widget and then resume playback from inside the app, the audio jumps back to approximately 1 minute into the session, even though I had been listening for 30 minutes.

I have noticed that pausing and resuming from inside the app seems to persist fine between listening sessions on different days.
